examples/asset/embedded_asset.rs

```diff
@@ -1,5 +1,7 @@
 //! Example of loading an embedded asset.
 
+// An embedded asset is an asset included in the program's memory, in contrast to other assets that are normally loaded from disk to memory when needed.
+// An embedded asset can reduce loading times since assets are immediately available, rather than being on disk.
```

Comment:
```suggestion
//! An embedded asset is an asset included in the program's memory, in contrast to other assets that are normally loaded from disk to memory when needed.
//! An embedded asset can reduce loading times since assets are immediately available, rather than being on disk.
```